dev-client/package.json

```diff
@@ -64,7 +64,7 @@
     "react-native-screens": "^3.32.0",
     "react-native-svg": "^15.3.0",
     "react-native-tab-view": "^3.5.2",
-    "terraso-client-shared": "github:techmatters/terraso-client-shared#f059d42",
+    "terraso-client-shared": "github:techmatters/terraso-client-shared#567f109",
```

Comment:
Note to self: Update this & npm install when upstream PRs get merged. Make sure package-lock actually updates.